content/sensu-go/5.15/api/federation.md

```diff
@@ -0,0 +1,374 @@
+
+### EXAMPLE {#etcd-replicatorsetcd-replicator-delete-example}
+
+The following example shows a request to delete the replicator `my_replicator`, resulting in a successful HTTP 204 No Content response.
+
+{{< highlight shell >}}
+curl -X DELETE \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+http://127.0.0.1:8080/api/enterprise/federation/v1/etcd-replicators/my_replicator
+
+HTTP/1.1 204 No Content
+{{< /highlight >}}
+
+#### API Specification {#etcd-replicatorsetcd-replicator-delete-specification}
+
+/etcd-replicators/:etcd-replicator (DELETE) | 
+--------------------------|------
+description               | Deletes the specified replicator from Sensu.
+example url               | http://hostname:8080/api/enterprise/federation/v1/etcd-replicators/{etcd-replicator_name}
+response codes            | <ul><li>**Success**: 204 (No Content)</li><li>**Missing**: 404 (Not Found)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+
+## The `/clusters` endpoint
+
+### `/clusters` (GET)
+
+The `/clusters` endpoint provides HTTP GET access to a list of clusters.
+
+#### EXAMPLE {#clusters-get-example}
+
+The following example demonstrates a request to the `/clusters` endpoint, resulting in a list of clusters.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/enterprise/federation/v1/clusters -H "Authorization: Bearer $SENSU_TOKEN"
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "us-west-2a"
+            },
+        "spec": {
+            "api_urls": [
+                "http://10.0.0.1:8080",
+                "http://10.0.0.2:8080",
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+#### API Specification {#clusters-get-specification}
+
+/clusters (GET)  | 
+---------------|------
+description    | Returns a list of clusters.
+example url    | http://hostname:8080/api/enterprise/federation/v1/clusters
+response type  | Array
+response codes | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+output         | {{< highlight shell >}}
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "us-west-2a"
+            },
+        "spec": {
+            "api_urls": [
+                "http://10.0.0.1:8080",
+                "http://10.0.0.2:8080",
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+## The `/clusters/:cluster` endpoint {#the-clusterscluster-endpoint}
+
+_**NOTE**: Only cluster admins have PUT and DELETE access to clusters._
+
+### `/clusters/:cluster` (PUT) {#clusterscluster-put}
+
+#### EXAMPLE {#clusterscluster-put-example}
+
+The following example demonstrates a request to update the cluster `us-west-2a`.
+
+{{< highlight shell >}}
+curl -X PUT \
```

Comment:
See an example [PUT](https://docs.sensu.io/sensu-go/5.14/api/checks/#checkscheck-put) for the correct arguments, and where to pass the request body.